style.css

```diff
@@ -1,818 +1,668 @@
 }
 
-/* about section */
+/* ---------------iNTRO-SECTION------------ */
 
-.about {
+.intro-container {
   display: flex;
-  flex-direction: column;
+  flex-direction: row;
+  justify-content: center;
+  background-color: #eeecec;
+  opacity: 0.96;
+  margin: 0;
 }
 
-.textAbout {
-  width: 90%;
-  height: fit-content;
-  font-family: 'Poppins', sans-serif;
-  font-size: 14px;
-  line-height: 21px;
-  color: #1a2236;
-  padding: 16px;
-  padding-bottom: 10px;
+.intro {
+  margin-top: 12%;
+  max-width: 70%;
+  margin-right: 0;
+  margin-left: 0;
+}
+
+.intro-title {
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  background-image: url('static/font-background.jpg');
+  font-size: 80px;
   margin-top: 0;
-  padding-top: 0;
+  margin-bottom: 0;
+  font-family: 'myFont', sans-serif;
 }
 
-.boxL {
-  padding: 16px;
-  font-family: 'Poppins', sans-serif;
-  display: flex;
-  flex-wrap: wrap;
-  align-content: center;
-  grid-auto-rows: 19.2rem;
-  grid-template-columns: repeat(auto-fill, minmax(22rem, 1fr));
+.year {
+  font-family: 'Lato', sans-serif;
+}
+
+.intro-greeting {
+  color: #dd610e;
+  margin-bottom: 0;
+  font-size: 35px;
+}
 
-  /* border: #1a2236 dotted 1px; */
+.intro-text {
+  border-style: solid;
+  border-color: #fff;
+  border-width: 2px;
+  padding: 20px 10px;
+  max-width: 500px;
+  line-height: 35px;
+  font-weight: 300;
 }
 
-.Box {
+.intro-date {
+  font-size: 40px;
+  margin-bottom: 10px;
+}
+
+.intro-place {
+  font-size: 25px;
+  margin-top: 0;
+  margin-bottom: 6%;
+}
+
+/* ---------------CARD-SECTION------------ */
+
+.program-container {
+  background-image: url('static/pjbck.jpg');
+  background-size: cover;
+  color: #fff;
   display: flex;
   flex-direction: column;
-  justify-content: center;
-  padding: 0;
-  padding-left: 50px;
-  padding-right: 50px;
-  height: 289px;
-  width: 289px;
-  margin: 9px auto;
+  align-items: center;
 }
 
-.lan {
-  background: #1c528c;
-  background-image: url("./images/lfs/hast.png");
-  background-repeat: no-repeat;
-  background-position: right bottom;
+.pr-section-title {
+  margin-top: 4%;
 }
 
-.fra {
-  background-color: #e2505c;
-  background-image: url("images/lfs/cuadrado.png");
-  background-repeat: no-repeat;
-  background-position: right bottom;
-  background-size: 33%;
+.pr-section-title::after {
+  content: "";
+  display: block;
+  width: 45px;
+  margin-left: 55px;
+  padding-top: 10px;
+  opacity: 0.7;
+  border-bottom: 3px solid #ec5242;
 }
 
-.ski {
-  background-color: #febf52;
-  background-image: url("images/lfs/foco.png");
-  background-repeat: no-repeat;
-  background-position: right bottom;
-  background-size: 35%;
+.card-container {
+  display: flex;
+  max-width: 75%;
+  gap: 3px;
+  margin-top: 4%;
 }
 
-h4 {
-  font-size: 20px;
-  font-weight: 800;
-  color: #fff;
+.card {
+  display: flex;
+  flex-direction: column;
+  gap: 3px;
+  align-items: center;
+  text-align: center;
+  background-color: rgba(100, 100, 100, 0.5);
 }
```

Comment:
Kindly ensure that the program section is as close to the design as possible. Note the following:

- All the cards have equal width
- A look at the design shows that the cards have a white border when they're hovered over
- The titles for the programs are not all uppercase.


Your design:
![image](https://github.com/Bleu-Yves-Sopoude/Bleu-Yves-Sopoude.github.io/assets/20554583/6e0f041a-34f1-4ca0-aec1-7559004781df)

Design Guide:
![image](https://github.com/Bleu-Yves-Sopoude/Bleu-Yves-Sopoude.github.io/assets/20554583/4c057917-006c-4a34-ae3c-a61854e6568c)